src/js/controllers/confirm.js

```diff
@@ -7,7 +7,6 @@ angular.module('copayApp.controllers').controller('confirmController', function(
   var countDown = null;
   var cachedSendMax = {};
   $scope.isCordova = platformInfo.isCordova;
-  $ionicConfig.views.swipeBackEnabled(false);
```

Comment:
Because some mobile user that were trying to do "swipe to pay" and it goes back to previous view.